src/utils/sqs_consumer.ts

```diff
@@ -0,0 +1,91 @@
+import { SQS } from 'aws-sdk';
+import delay from 'delay';
+
+import { ONE_SECOND_MS } from '../constants';
+import { logger } from '../logger';
+
+import { SqsClient } from './sqs_client';
+
+export type MessageHandler = (message: SQS.Types.Message) => Promise<any>;
```

Comment:
Can avoid the `any` here (may or may not be worth it):
```
export type MessageHandler<T> = (message: SQS.Types.Message) => Promise<T>;
```

Reply:
I like this!

Reply:
ah, but sadly, doing this will force `T` to cascade through the Sqs Consumer, which is a bigger change. 

I will keep this in the back of my head though, if we ever need typesafety here, we should do it